client.lua

```diff
@@ -1542,6 +1543,7 @@ RegisterNUICallback('removeComponent', function(data, cb)
 
 					if success then
 						RemoveWeaponComponentFromPed(playerPed, currentWeapon.hash, component)
+						TriggerEvent('ox_inventory:updateWeaponComponents', "removed", currentWeapon, component)
```

Comment:
`ox_inventory:updateWeaponComponent`.
Use `'` rather than `"` (i.e. 'removed', 'added').
Send the item name of the component being used (not just the hash).
Don't think there's much need to send currentWeapon at all.

Reply:
Done in latest commit, but double check it. I'll also PR 3095 weapons and components later

Reply:
```lua
TriggerEvent('ox_inventory:updateWeaponComponent', action --[['removed' | 'added']], component --[[hash]], componentItem --[[string]])
```